### PR TITLE
Training Script with proper logging

### PR DESCRIPTION
**Pull request by lucky-suman**
All the jobs(Reading data from S3, Training, and saving model and hyperparameters on S3 bucket).

### CODE REVIEW

**Comment by sunilk747** [comment]
LGTM :+1:

File: training/train.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    s3_object.connect()
+    logger.info("S3 connection established.")
+    return s3_object
+ 
+def load_data(s3_client):
+    HPF_output_package_id_dict = os.path.join("maven", DEPLOYMENT_PREFIX, 
+                                              MODEL_VERSION, "trained-model/package_id_dict.json")
+    HPF_output_manifest_id_dict = os.path.join("maven", DEPLOYMENT_PREFIX, 
+                                               MODEL_VERSION, "trained-model/manifest_id_dict.json")
+    print("Calculated")
```
Comment (miteshvp) [inline training/train.py]: Remove the debug

File: training/train.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    if ((s3_client.object_exists(HPF_output_package_id_dict)) and
+        (s3_client.object_exists(HPF_output_manifest_id_dict))):
+        print("entered")
```
Comment (miteshvp) [inline training/train.py]: same as above.

File: training/train.py (hunk 1 above)
Comment (lucky-suman) [reply training/train.py]: Done

File: training/train.py (hunk 2 above)
Comment (lucky-suman) [reply training/train.py]: Done

**Pull request merged**